src/components/application_manager/rpc_plugins/rc_rpc_plugin/test/commands/get_interior_vehicle_data_request_test.cc

```diff
@@ -180,37 +178,52 @@ TEST_F(GetInteriorVehicleDataRequestTest,
               ManageHMICommand(HMIResultCodeIs(
                   hmi_apis::FunctionID::RC_GetInteriorVehicleData)))
       .WillOnce(Return(true));
+  // Act
+  command->Run();
+}
 
+TEST_F(GetInteriorVehicleDataRequestTest,
+       Execute_ExpectCorrectMessageSentToHMI_SupscribeAppNoCache) {
+  // Arrange
+  InitialExpectations();
+  auto module_type = mobile_apis::ModuleType::RADIO;
```

Comment:
@LitvinenkoIra It's seems better to move `module_type` in GetInteriorVehicleDataRequestTest and make const. I see that we are using `mobile_apis::ModuleType::RADIO` in all tests. We can set this value once in class.

Reply:
@ZhdanovP fixed
fe38fb0